modules/statistics/AvgRTPStatsReporter.js

```diff
@@ -452,8 +565,6 @@ export default class AvgRTPStatsReporter {
         this._avgRemoteFPS.reset();
         this._avgLocalFPS.reset();
         this._avgCQ.reset();
-        this._avgRTT.reset();
-        this._avgRemoteRTTMap.clear();
```

Comment:
It looks like this function fails to reset _avgPacketLossTotal. Should it also reset jvbStatsMonitor and p2pStatsMonitor?